lib/suse/connect/yast.rb

```diff
@@ -7,6 +7,19 @@ module Connect
     class YaST
       class << self
 
+        # Returns a hash containing default configuration values.
+        # Keys: :config_file_path, :scc_url, :server_cert_file_path, :update_certificates_script_path, :credentials_dir_path, :global_credentials_file_path
+        # @return [Hash]
+        def defaults
```

Comment:
I am not sure if you can do B_CONST = A_CONST when A_CONST was defined in another class. If this works it would be a good solution.